tools/hooks/requirements_changed.rb

```diff
@@ -0,0 +1,34 @@
+REPO_DIR = File.expand_path('../../../', __FILE__)
+
+REQUIREMENTS = {
+  "requirements.txt" => "pip install -r requirements.txt",
```

Comment:
we don't actually use this right? No big deal since it should be a no-op if that is the case.